Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java

```diff
@@ -411,6 +411,7 @@ protected void onPause()
   protected void onStop()
   {
     super.onStop();
+    Settings.manageGCAdapterScanThread();
```

Comment:
Why is this called here?

Reply:
If the GCAdapter is used when emulation starts and then changed to a different controller, the GCAdapter scanning thread may no longer be needed. Selecting the GCAdapter outside of emulation will restart the scanning thread if needed. GCAdapter changes during emulation (which currently aren't an option) will be handled by #8894.

Reply:
I should probably move this to after making a Wii Overlay controller change.

Reply:
I don't get it, though. If GCAdapter changes during emulation aren't an option, how could it be changed into a different controller during emulation? And why should the thread stop in `onStop` instead of as soon as the controller changes?

Reply:
You can't select the GameCube Adapter during emulation but you can change the controller to something else (like a Classic Controller), making the scanning thread unnecessary. Agreed on the last sentence.

Reply:
Currently, changing the controller doesn't update `m_SIDevice`. This is a more complicated change so I'd rather handle this in  #8894 now.